mlir/test/Gradient/BufferizationTest.mlir

```diff
@@ -28,26 +28,45 @@ func.func @adjoint(%arg0: f64, %arg1: index) {
     return
 }
 
-///
+// -----
 
 func.func private @circuit2(%arg0: f64)
 
 func.func @backprop(%arg0: f64, %arg1: index) {
```

Comment:
It's good to match the function name first as follows before matching its contents in these type of tests:
```suggestion
// CHECK-LABEL: @backprop
func.func @backprop(%arg0: f64, %arg1: index) {
```